packages/openai/src/openai-completion-language-model.test.ts

```diff
@@ -26,13 +55,22 @@ describe('doGenerate', () => {
       total_tokens: 34,
       completion_tokens: 30,
     },
+    logprobs = null,
+    finish_reason = 'stop',
   }: {
     content?: string;
     usage?: {
       prompt_tokens: number;
       total_tokens: number;
       completion_tokens: number;
     };
+    logprobs?: {
+      tokens: string[];
+      token_logprobs: number[];
+      top_logprobs: Record<string, number>[];
+      text_offset: number[];
```

Comment:
is `text_offset` needed? if we don't use it, i'd prefer to not leave it out (to simplify the code, reduce zod validation time, and increase compatibility with not-quite-so-compatible 3rd party apis)